internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -438,10 +417,10 @@ func (r *BMACReconciler) reconcileBMH(ctx context.Context, bmh *bmh_v1alpha1.Bar
 			}
 
 			if infraEnv.Status.ISODownloadURL == "" {
-				// the image has not been created yet, try later.
-				r.Log.Infof("Image URL for InfraEnv (%s/%s) not available yet. Retrying reconcile for BareMetalHost  %s/%s",
+				// the image has not been created yetlater.
```

Comment:
Was this change intentional?

Reply:
it was not, merge conflict gone wrong! :D